### PR TITLE
DGX A100 CVEs - June 2022

### PR DESCRIPTION
**Pull request by PSIRT-NVIDIA**
DGX A100 CVEs - June 2022

### CODE REVIEW

File: 2022/28xxx/CVE-2022-28200.json [hunk 1]
```diff
@@ -1,18 +1,68 @@
 {
```
Comment (cve-team) [inline 2022/28xxx/CVE-2022-28200.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/28xxx/CVE-2022-28200.json:1]: ("Expecting ',' delimiter: line 68 column 6 (char 2168)",)

File: 2022/31xxx/CVE-2022-31599.json [hunk 2]
```diff
@@ -1,18 +1,68 @@
 {
```
Comment (cve-team) [inline 2022/31xxx/CVE-2022-31599.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/31xxx/CVE-2022-31599.json:1]: ("Expecting ',' delimiter: line 68 column 6 (char 2115)",)

File: 2022/31xxx/CVE-2022-31600.json [hunk 3]
```diff
@@ -1,18 +1,68 @@
 {
```
Comment (cve-team) [inline 2022/31xxx/CVE-2022-31600.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/31xxx/CVE-2022-31600.json:1]: ("Expecting ',' delimiter: line 68 column 6 (char 2170)",)

File: 2022/31xxx/CVE-2022-31601.json [hunk 4]
```diff
@@ -1,18 +1,68 @@
 {
```
Comment (cve-team) [inline 2022/31xxx/CVE-2022-31601.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/31xxx/CVE-2022-31601.json:1]: ("Expecting ',' delimiter: line 68 column 6 (char 2049)",)

File: 2022/31xxx/CVE-2022-31602.json [hunk 5]
```diff
@@ -1,18 +1,68 @@
 {
```
Comment (cve-team) [inline 2022/31xxx/CVE-2022-31602.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/31xxx/CVE-2022-31602.json:1]: ('Invalid control character at: line 37 column 27 (char 1102)',)

File: 2022/31xxx/CVE-2022-31603.json [hunk 6]
```diff
@@ -1,18 +1,68 @@
 {
```
Comment (cve-team) [inline 2022/31xxx/CVE-2022-31603.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/31xxx/CVE-2022-31603.json:1]: ("Expecting ',' delimiter: line 68 column 6 (char 2112)",)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/31xxx/CVE-2022-31599.json (hunk 2 above)
Comment (cve-team) [inline 2022/31xxx/CVE-2022-31599.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/31xxx/CVE-2022-31599.json:1]: ("Expecting ',' delimiter: line 68 column 6 (char 2115)",)

File: 2022/31xxx/CVE-2022-31600.json (hunk 3 above)
Comment (cve-team) [inline 2022/31xxx/CVE-2022-31600.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/31xxx/CVE-2022-31600.json:1]: ("Expecting ',' delimiter: line 68 column 6 (char 2170)",)

File: 2022/31xxx/CVE-2022-31601.json (hunk 4 above)
Comment (cve-team) [inline 2022/31xxx/CVE-2022-31601.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/31xxx/CVE-2022-31601.json:1]: ("Expecting ',' delimiter: line 68 column 6 (char 2049)",)

File: 2022/31xxx/CVE-2022-31602.json (hunk 5 above)
Comment (cve-team) [inline 2022/31xxx/CVE-2022-31602.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/31xxx/CVE-2022-31602.json:1]: ('Invalid control character at: line 37 column 27 (char 1102)',)

File: 2022/31xxx/CVE-2022-31603.json (hunk 6 above)
Comment (cve-team) [inline 2022/31xxx/CVE-2022-31603.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/31xxx/CVE-2022-31603.json:1]: ("Expecting ',' delimiter: line 68 column 6 (char 2112)",)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by PSIRT-NVIDIA** [comment]
Will resubmit new request.

**Pull request closed without merging**